src/runtime/constructorbinding.cs

```diff
@@ -19,18 +19,20 @@ namespace Python.Runtime
     /// and creating the BoundContructor object which contains ContructorInfo object.
     /// 3) In tp_call, if ctorInfo is not null, ctorBinder.InvokeRaw() is called.
     /// </remarks>
+    [Serializable]
     internal class ConstructorBinding : ExtensionType
     {
         private Type type; // The managed Type being wrapped in a ClassObject
         private IntPtr pyTypeHndl; // The python type tells GetInstHandle which Type to create.
         private ConstructorBinder ctorBinder;
+
+        [NonSerialized]
```

Comment:
It just a cache object of str, it would be recreate if it not exists. Well, it means you have to implement OnSave/OnLoad for it if it needs to be serialized.